server/proxy/contexts.go

```diff
@@ -47,15 +47,65 @@ func (cp *contextsProxy) List(ctx context.Context, request *contextsv1.ListReque
 		return &contextsv1.ListResponse{}, err
 	}
 
+	return convertContexts(contexts, configFile.CurrentContext), nil
+}
+
+func convertContexts(contexts []*store.DockerContext, currentContext string) *contextsv1.ListResponse {
 	result := &contextsv1.ListResponse{}
 
 	for _, c := range contexts {
-		result.Contexts = append(result.Contexts, &contextsv1.Context{
+		endpointName := c.Type()
+		if c.Type() == store.DefaultContextType {
+			endpointName = "docker"
```

Comment:
Why not default?

Reply:
to accommodate existing structure (mapped to `docker context ls` json output BTW): endpoints are in a dictionary, the key is `docker` for the host, and context type for other contexts.